package.json

```diff
@@ -12,26 +12,29 @@
     "url": "https://github.com/Microsoft/ui-fabric-react-native.git"
   },
   "scripts": {
-    "preinstall": "node ./scripts/use-yarn-please.js",
-    "postinstall": "node ./scripts/postinstall.js",
+    "build": "lage build",
+    "build-tools": "lage build-tools",
+    "buildci": "lage buildci",
+    "bump-versions": "beachball bump",
     "clean-all": "node ./scripts/clean-all.js",
-    "start": "cd apps && cd demo && npm start",
     "docs": "yarn workspace fluent-rn-website start",
-    "build": "lerna run build --stream -- --production --lint --color",
-    "bundle": "lerna run bundle --stream -- --production --lint --color",
+    "bundle": "lage bundle",
     "clean": "lerna run clean --stream",
```

Comment:
Why is the command still using `lerna`?

Reply:
I haven't removed lerna, some of the magic build stuff intuits repo structure using the lerna config but this is a good catch. Honestly it only uses the config so I could actually replace all the commands.

Reply:
Fixed